proxy/envoy/config.go

```diff
@@ -128,9 +120,10 @@ func buildListeners(context *ProxyContext) (Listeners, Clusters) {
 	outbound, outClusters := buildOutboundListeners(instances, services, context)
 
 	listeners := append(inbound, outbound...)
-	listeners.normalize()
+	clusters := append(inClusters, outClusters...)
 
-	clusters := append(inClusters, outClusters...).normalize()
+	listeners = listeners.normalize()
```

Comment:
Yes, but it's a better style to split two operations into two statements I think, so I left it in this PR. Code looks cleaner.